pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -718,6 +733,10 @@ func ValidateNodeGroup(i int, ng *NodeGroup) error {
 		}
 	}
 
+	if len(ng.LocalZones) > 0 && len(ng.AvailabilityZones) > 0 {
+		return errors.New("cannot specify both localZones and availabilityZones")
```

Comment:
How come? Can't you have a hybrid cluster with specific workloads running in a localzone vs some other nodegroup which defines availability zones that will take core a different workload?

Reply:
Oh wait, never mind. This is the same nodegroup.

Reply:
>Can't you have a hybrid cluster with specific workloads running in a localzone vs some other nodegroup which defines availability zones that will take core a different workload?

You can. As you noticed, this validation is to disallow nodegroups having both local zones and AZs defined.